site-content/source/modules/ROOT/pages/blog/Apache-Cassandra-Changelog-9-August-2021.adoc

```diff
@@ -44,7 +44,7 @@ We are pleased to announce that *Jon Meredith* has been invited by the Project M
 
 === Added
 
-With the release of Apache Cassandra 4.0, we have established a https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=484.[New Release tracking Kanboard,window=_blank]. For those just getting started with the project, there is a "https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=484&quickFilter=2162&quickFilter=2160[Starter Tickets,window=_blank]" quick label that corresponds to our Low Hanging Fruit status. Any of these tickets should be of appropriate complexity for someone new to the project to take on.
+With the release of Apache Cassandra 4.0, we have established a https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=484[New Release tracking Kanboard,window=_blank]. For those just getting started with the project, there is a https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=484&quickFilter=2162&quickFilter=2160["Starter Tickets",window=_blank] quick label that corresponds to our Low Hanging Fruit status. Any of these tickets should be of appropriate complexity for someone new to the project to take on.

```

Comment:
This is already fixed on the site by [CASSANDRA-16980](https://issues.apache.org/jira/browse/CASSANDRA-16980) (PR #73).